api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/MessagePayloadFactory.java

```diff
@@ -49,23 +49,39 @@ public static MakePaymentMessagePayload buildMakePaymentMessagePayload(PaymentCy
      * @return The constructed payload.
      */
     public static EmailMessagePayload buildSendNewCardSuccessEmailPayload(PaymentCycle paymentCycle) {
+        Map<String, Object> emailPersonalisation = createPaymentEmailPersonalisationMap(paymentCycle);
+        return EmailMessagePayload.builder()
+                .claimId(paymentCycle.getClaim().getId())
+                .emailType(EmailType.NEW_CARD)
+                .emailPersonalisation(emailPersonalisation)
+                .build();
+    }
+
+    public static EmailMessagePayload buildPaymentNotificationEmailPayload(PaymentCycle paymentCycle) {
+        Map<String, Object> emailPersonalisation = createPaymentEmailPersonalisationMap(paymentCycle);
+        return EmailMessagePayload.builder()
+                .claimId(paymentCycle.getClaim().getId())
+                .emailType(EmailType.PAYMENT)
+                .emailPersonalisation(emailPersonalisation)
+                .build();
+    }
+
+    private static Map<String, Object> createPaymentEmailPersonalisationMap(PaymentCycle paymentCycle) {
         Claimant claimant = paymentCycle.getClaim().getClaimant();
         PaymentCycleVoucherEntitlement voucherEntitlement = paymentCycle.getVoucherEntitlement();
         Map<String, Object> emailPersonalisation = new HashMap<>();
         emailPersonalisation.put(EmailTemplateKey.FIRST_NAME.getTemplateKeyName(), claimant.getFirstName());
         emailPersonalisation.put(EmailTemplateKey.LAST_NAME.getTemplateKeyName(), claimant.getLastName());
-        emailPersonalisation.put(EmailTemplateKey.FIRST_PAYMENT_AMOUNT.getTemplateKeyName(),
-                convertPenceToPounds(paymentCycle.getTotalEntitlementAmountInPence()));
+        // TODO: MGS: HTBHF-2282 remove FIRST_PAYMENT_AMOUNT once the instant success template has been updated to use PAYMENT_AMOUNT
```

Comment:
I thought the templates were ready to go? I don't like working on these changes when the template isn't 100% sure, I'd have kicked this back to Sam/Laura before starting if that was the case.

Reply:
I was thinking we should change the instant success template (already in use) so that it uses 'payment_amount' instead of 'first_payment_amount' to be consistent with other templates. One alternative is to specify both, as we're doing here. Another is to have different methods populating the message payload for different notify templates.